src/main.js

```diff
@@ -185,6 +186,9 @@ function run(commands) {
         if (IS_WINDOWS) {
             spawnOpts.detached = false;
         }
+        if (supportsColor) {
```

Comment:
I opted for this if statement rather than what we talked about in the issue because it makes the intent a little more clear.